os/include/tinyara/reboot_reason.h

```diff
@@ -40,6 +40,10 @@ typedef enum {
 	REBOOT_SYSTEM_BINARY_UPDATE        = 7,	 /* Reboot for Binary Update */
 	REBOOT_SYSTEM_BINARY_RECOVERYFAIL  = 8,	 /* Binary Recovery Fail */
 
+	REBOOT_SYSTEM_DSLP_RESET           = 27, /* System wake up from deep sleep */
+	REBOOT_SYSTEM_SYS_RESET            = 28, /* System reset by calling sys_reset */
```

Comment:
Is there any case for calling `sys_reset` from bootloader?
Within TizenRT side, we all add the reboot reason when calling `sys_reset`.

Reply:
Not from bootloader.
But from KM0 (dlp), a prebuild image provided by RTK. (km0_image2_all.bin)

Reply:
Okay. Thanks for checking.
`sys_reset` is only for RTK, not general name, but this is common header.
How about changing the comment like `system reset by core` or something which can explain the case?

Reply:
Done change.